src/gui/mrview/mode/base.cpp

```diff
@@ -192,7 +192,7 @@ namespace MR
           const Projection* proj = get_current_projection();
           if (!proj) return;
           set_target (target() - proj->screen_to_model_direction (window().mouse_displacement(), target()));
-          updateGL();
+          // updateGL(); # updateGL() causes pan gestures to remain in state Qt::GestureUpdated, never reaching Qt::GestureFinished on macOS
```

Comment:
Happy to remove if no side effects - but I think you'd be better off deleting the line altogether, and placing the comment you've added in the git history. I'd rather keep the code free of unnecessary legacy, and document the rationale for changes in the history - that's what it's for... :wink: